src/explore-education-statistics-admin/src/pages/publication/components/__tests__/PublicationAddExistingUsersTab.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import PublicationAddExistingUsersTab from '@admin/pages/publication/components/PublicationAddExistingUsersTab';
+import _releasePermissionService, {
+  ManageAccessPageContributor,
+} from '@admin/services/releasePermissionService';
+
+jest.mock('@admin/services/releasePermissionService');
+
+describe('PublicationAddExistingUsersTab', () => {
+  const testContributors: ManageAccessPageContributor[] = [
+    {
+      userId: 'user-1',
+      userFullName: 'User Name 1',
+      userEmail: 'user1@test.com',
+      releaseRoleId: 'release-role-1',
+    },
+    {
+      userId: 'user-2',
+      userFullName: 'User Name 2',
+      userEmail: 'user2@test.com',
+      releaseRoleId: undefined,
+    },
+    {
+      userId: 'user-3',
+      userFullName: 'User Name 3',
+      userEmail: 'user3@test.com',
+      releaseRoleId: 'release-role-3',
+    },
+    {
+      userId: 'user-4',
+      userFullName: 'User Name 4',
+      userEmail: 'user4@test.com',
+      releaseRoleId: undefined,
+    },
+  ];
```

Comment:
We should at least have a test to check that we can't submit an empty list of users (by adding some validation to the form).